web/src/hooks/useTodoAPI.tsx

```diff
@@ -0,0 +1,14 @@
+import { useContext, useEffect, useState } from 'react'
+import { AuthContext } from 'react-oauth2-code-pkce'
+import TodoAPI from '../api/TodoAPI'
+
+export function useTodoAPI() {
```

Comment:
Since this is the only export in this file, maybe use `export default`?

```suggestion
export default function useTodoAPI() {
```

If changed, remember to update the import from `import { useTodoAPI }` to `import useTodoAPI`.

Reply:
@sebastianvitterso what are the benefits?

Reply:
I just always liked the idea of being able to write `import myThing from './myThing'` - I'm not sure if there are any real benefits. 

In fact - there might be a drawback by having the developer switch between and consider `export` or `export default`. 

... 

I just always thought that default imports should match the file's name, and be the "main thing" they're exporting. So if we assume that this hook has a few exported constants (e.g. `export const TOKEN = 'abc123'`, as well as the hook itself, I would assume that the hook was defaultually exported. 

Disregard my comment for now, but I always considered default imports a slightly cleaner way to import stuff.

Reply:
I found a few (but not _very_ good) reasons to why [default is bad](https://basarat.gitbook.io/typescript/main-1/defaultisbad), but I couldn't find very good reasons to use default either. I guess you could have some benefits from bundling if there are unused exports in the file? Not sure if it would make a difference as long as there's only one export. 🤔 Anyway, thanks for learning me more about default exports. If you want to use default later, maybe we can add an eslint rule for it, and get a some help reminding us to use it? 😇